app/controllers/ember-community-survey-2017.js

```diff
@@ -391,118 +341,88 @@ const longLived = {
         15.8, // '3–6 Months',
         19.8, // '< 3 Months',
       ],
-      color: emberOrange,
     },
   ],
-};
+}).highchartsOptions;
 
-const serverLanguage = {
-  options: {
-    chart: {
-      type: 'column',
-    },
-    title: {
-      text: 'What is the primary language of your server-side stack?',
-    },
-    subtitle: {
-      text: '',
-    },
-    xAxis: {
-      type: 'category',
-    },
-    yAxis: {
-      title: {
-        text: 'Percent',
-      },
-    },
+const serverLanguage = new VerticalBarChart({
+  chart: {
+    categories: [
+      'Ruby',
+      'JavaScript',
+      'Java',
+      'Python',
+      'PHP',
+      '.NET',
+      'Elixir / Erlang',
+      'Go',
+      'Scala',
+      'TypeScript',
+      'C++',
+      'Other',
+    ],
+    title: 'What is the primary language of your server-side stack?',
   },
-  data: [
+
+  rawData: [
     {
-      name: '2016',
       color: darkGrayColor,
-      data: [
-        { y: 41, name: 'Ruby' },
-        { y: 24, name: 'JavaScript' },
-        { y: 17, name: 'Java' },
-        { y: 13, name: 'PHP' },
-        { y: 12, name: 'Python' },
-        { y: 8, name: '.NET' },
-        { y: 6, name: 'Elixir / Erlang' },
-        { y: 5, name: 'Go' },
-        { y: 2, name: 'Scala' },
-        { y: 0, name: 'TypeScript' },
-        { y: 0, name: 'C++' },
-        { y: 1, name: 'Rust' },
-        { y: 3, name: 'Other' },
-      ],
+      label: '2016',
+      values: [41, 24, 17, 13, 12, 8, 6, 5, 2, 0, 0, 1, 3],
```

Comment:
Before the changes, the `data` arrays for 2016 and 2017 didn't list the categories in the same order (e.g. PHP and Python), so we unfortunately want to be careful with creating tuples. Luckily, we have Percy to tell us if we might have switched an order.

```suggestion
      values: [41, 24, 17, 12, 13, 8, 6, 5, 2, 0, 0, 1, 3],
```